server/server.c

```diff
@@ -0,0 +1,111 @@
+#include "server.h"
+
+void socketThread(int clientSocket)
+{
+    char client_message[2000];
```

Comment:
Client message is 2000, server buffer is 1024 and client buffer is 512.
This is confusing. They should all be the same size.

Reply:
changed to #define BUFF_SIZE 2048